packages/autotask-client/src/api.ts

```diff
@@ -108,4 +113,10 @@ export class AutotaskClient extends BaseApiClient {
       return await api.put(`/autotasks/${autotaskId}/code`, { encodedZippedCode });
     });
   }
+
+  public async createSecrets(data: SaveSecretsRequest): Promise<GetSecretsResponse> {
+    return this.apiCall(async (api) => {
+      return await api.post(`/autotasks/secrets`, data);
```

Comment:
```suggestion
      return await api.post(`/secrets`, data);
```

You shouldn't need the leading `/autotasks` here, that's provided in the base `DEFENDER_AUTOTASK_API_URL`

Reply:
Yep, API gateway mapping confused me there for a sec.